packages/mds-agency/handlers/create-event.ts

```diff
@@ -50,28 +50,28 @@ const handleDbError = async (
  * @param event VehicleEvent which was persisted
  * @param deltaThreshold Delta (in ms) to log for (default 100ms)
  */
-const logEventWritePerformance = (event: VehicleEvent, logThreshold = 100) => {
+const logEventWritePerformance = async (event: VehicleEvent, logThreshold = 100) => {
```

Comment:
In these cases I question the value of including the provider name in the log. To me, it’s not worth the cost when we could just log the id.